etl/steps/data/grapher/worldbank_wdi/2025-01-24/wdi.meta.yml

```diff
@@ -0,0 +1,524 @@
+definitions:
+  # NOTE: we'd like to move this to snapshot, but it's impractical because there are
+  # tons of other downstream datasets that still use sources
+  wdi_origin: &wdi_origin
```

Comment:
Should we still use this? If so, we need to update the metadata

Reply:
I'm going to tackle this as part of fixing origins for all indicators https://github.com/owid/etl/issues/3609.